fonts.css

```diff
@@ -1,82 +1,410 @@
-		 url("C:\Users\пк\Desktop\Fonts\montserrat-medium-webfont.woff2") format("woff2");
-	font-weight: 500;
-	font-style: normal;
-}
-
-@font-face {
-    font-family: "Montserrat";
-    src: url("C:\Users\пк\Desktop\Fonts\montserrat-semibold-webfont.woff2") format("woff2"),
-         url("C:\Users\пк\Desktop\Fonts\montserrat-semibold-webfont.woff") format("woff");
-    font-weight: 600;
-    font-style: normal;
-}
+/* Шрифты */
+	@font-face {
+		font-family: "Montserrat";
+		src:	url("Fonts/Montserrat-Bold.otf") format("opentype"),
+					url("Fonts/Montserrat-Bold.ttf") format("truetype");
+		font-weight: 700;
+		font-style: normal;
+	}
+
+	@font-face {
+		font-family: "Montserrat";
+		src: url("Fonts/Montserrat-Medium.otf") format("opentype"),
+				 url("Fonts/Montserrat-Medium.ttf") format("truetype");
+		font-weight: 500;
+		font-style: normal;
+	}
+
+	@font-face {
+	  font-family: "Montserrat";
+	  src: url("Fonts/Montserrat-SemiBold.otf") format("opentype"),
+				 url("Fonts/Montserrat-SemiBold.ttf") format("truetype");
+	  font-weight: 600;
+	  font-style: normal;
+	}
 
-@font-face {
+	@font-face {
     font-family: "Montserrat";
-    src: url("C:\Users\пк\Desktop\Fonts\montserrat-regular-webfont.woff2") format("woff2"),
-         url("C:\Users\пк\Desktop\Fonts\montserrat-regular-webfont.woff") format("woff");
-    font-weight: 400;
-    font-style: normal;
-}
+    src: url("Fonts/Montserrat-Regular.otf") format("opentype"),
+		 		 url("Fonts/Montserrat-Regular.ttf") format("truetype");
+	  font-weight: normal;
+	  font-style: normal;
+	}
 
-/* Шапка */
+	body {
+		font-family: "Montserrat";
+	}
 
-.header__link {
-	font-family: "Montserrat";
-	font-weight: 600;
-}
 
-.header__wrapper {
-	font-family: "Montserrat";
-	font-weight: 600;
-}
+/* Desktop */
+	
+	/* Шапка */
+		.header {
+			font-weight: 600;
+			font-size: 15px;
+			line-height: 18px;
+		}
 
-/* Домашний экран */
+	/* Домашний экран */
+		.section-main__title {
+			font-weight: 600;
+			font-size: 90px;
+			line-height: 110px;
+		}
 
-.character__item_big {
-	font-family: "Montserrat";
-	font-weight: 600;
-}
+		.section-main__subtitle {
+			font-weight: 500;
+			font-size: 15px;
+			line-height: 18px;
+		}
 
-.character__item_small {
-    font-family: "Montserrat";
-	font-weight: 400;
-}
-
-.btn__item_link {
-	font-family: "Montserrat";
-	font-weight: 700;
-}
-
-.button__item_link {
-	font-family: "Montserrat";
-	font-weight: 700;
-}
-
-/* Третий экран*/
-
-.star {
-	font-family: "Montserrat";
-	font-weight: 700;
-}
-
-.score {
-	font-family: "Montserrat";
-	font-weight: 600;
-}
-
-.safe-body {
-	font-family: "Montserrat";
-	font-weight: 600;
-}
\ No newline at end of file
+		.character__item_big {
+			font-weight: 600;
+			font-size: 35px;
+			line-height: 40px;
+		}
+
+		.character__item_little {
+			font-size: 20px;
+			line-height: 24px;
+		}
+
+		.character__item_small {
+			font-weight: normal;
+			font-size: 12px;
+			line-height: 15px;
+		}
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)